apps/explorer/lib/explorer/chain.ex

```diff
@@ -172,6 +172,24 @@ defmodule Explorer.Chain do
     end
   end
 
+
+  @doc """
+  Estimated count of `t:Explorer.Chain.SmartContracts.t/0`.
+
+  Estimated count of contract addresses.
+  """
+  @spec contracts_estimated_count() :: non_neg_integer()
+  def contracts_estimated_count do
+    cached_value = ContractCounter.fetch()
+
+    if is_nil(cached_value) do
+     %Postgrex.Result{rows: [[count]]} = Repo.query!("SELECT COUNT(*) FROM smart_contracts;")
```

Comment:
This can only be counted if the contract is verified, and the contract without verification cannot be counted. you should use 'select count(created_contract_address_hash) from transactions where created_contract_address_hash != '';'